scripts/validate_snapshots.py

```diff
@@ -65,7 +65,7 @@ def main(args):
                     result = tdr_plugin._run_sql(query)
                     for row in result:
                         match = {
-                            'catalog': args.catalog,
```

Comment:
i think this is more than a typo. If I understand the code, the intent here to print some diagnostic information indicating where a problematic string was found. If the script is processing more than one catalog, that diagnostic should only include the _current_ catalog, not a list of every catalog passed to the script.